### PR TITLE
blackflag units after wars

### PR DESCRIPTION
**Pull request by wxwisiasdf**

### CODE REVIEW

**Comment by wxwisiasdf** [comment]
fixes issue were ai wouldnt move units after wars

File: src/military/military.cpp [hunk 1]
```diff
@@ -2599,6 +2599,8 @@ void cleanup_war(sys::state& state, dcon::war_id w, war_result result) {
 	}
 
 	state.world.delete_war(w);
+
+	update_blackflag_status(state);
```
Comment (schombert) [inline src/military/military.cpp:2603]: this won't actually do anything (see the details of the function). It is also redundant  -- as part of the war ending nations are removed from it. That triggers updating the blag flag status of all armies later in the day

Comment (schombert) [reply src/military/military.cpp:2603]: I do see that I forgot to put the call to this function in after the military gc runs. That probably explains the issue you are seeing. Let me push that up and see if it fixes things for you

Comment (wxwisiasdf) [reply src/military/military.cpp:2603]: doesnt seem to be the case through

**Comment by wxwisiasdf** [comment]
seems kosher now

**Pull request closed without merging**